### PR TITLE
Adds PostalAddress PUT

### PR DESCRIPTION
**Pull request by carlrc**
## PR Type
What kind of change does this PR introduce?
```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Tests
[ ] Other
```

## Checklist
Have you done the following?
```
[X] Exported any new types in index.ts
[X] Merged master
[X] Run `npm run lint`
[X] Ensure the tests pass
```

### CODE REVIEW

File: src/BrightpearlClient.ts [hunk 1]
```diff
@@ -199,6 +200,19 @@ class BrightpearlClient {
...
+        const elementsToReplace: ElementsToReplace[] = [
+            {
+                key: CONTACT_ID,
+                value: contactId.toString()
+            }
+        ];
+        const url = UrlHelper.getUrl(`${this.baseUrl}/${ENDPOINTS.contact.postal_address.put}`, elementsToReplace);
+        await this.axios.put(url,
+            postalAddresses,
+            { headers: this.getHeaders()});
```
Comment (grahamcarpenter) [inline src/BrightpearlClient.ts]: There is a space after the `{` but not before the `}`. Looks like it's the same for the other methods, but it kind of bothers me.

File: src/Endpoints.ts [hunk 2]
```diff
@@ -19,9 +19,10 @@ const BRAND_ID = "brandId";
 const ENDPOINTS  = Object.freeze({
     contact: {
         search: `${CONTACT}/contact-search`,
-        get: `${CONTACT}/contact/{${ID}}`,
+        get: `${CONTACT}/contact/{${CONTACT_ID}}`,
```
Comment (grahamcarpenter) [inline src/Endpoints.ts]: 👍

**Comment by KumarShyam** [comment]
To avoid the re-work : I can include this PR as a new merge commit in my upcoming PR for Product Integration. I have done same changes as part of Contact - Postal Address interfaces and PUT method API.
This will help in not spending additional time on merge conflicts.

File: src/BrightpearlClient.ts (hunk 1 above)
Comment (carlrc) [reply src/BrightpearlClient.ts]: fixed them

**Pull request merged**